### ISSUE TITLE
Losing existing local def. of profile on pull

### ISSUE THREAD

**Issue by Confirm4Crit** [forcedotcom/cli#509]
### Summary

While using push/pull against a sandbox, running pull ends up getting a copy of the touched profiles and FLS that was touched, nothing else. Existing properties on the local profile are deleted, as if it was an mdapi pull of that profile and the touched field. 

I'd lose fls, apex class permissions, and other props already on the profile locally.
### Steps To Reproduce:
(I haven't been able to cleanly do this with a scratch org, either, none of the trailhead apps have profiles from what I saw.)

Expected:
Create a field or tweak a profile in other ways.
Run pull.
Those changes should get pulled without nuking the rest of the profile.

### Actual result

It seems like an MDAPI call is made of X profiles, X *changed* fields, and other things, layouts, etc, which causes existing FLS on the profile locally to be removed.
![image](https://user-images.githubusercontent.com/3483618/88204768-c6d02580-cc19-11ea-9cf0-4e90552c824f.png)

### Additional information

The definition of the profile in my vcs is using https://github.com/Accenture/sfpowerkit#sfpowerkitsourceprofileretrieve
We aren't on perm sets yet, and use profiles for FLS. I know that's a nono.

**SFDX CLI Version**(to find the version of the CLI engine run sfdx --version):
sfdx-cli/7.66.2 darwin-x64 node-v14.5.0
**SFDX plugin Version**(to find the version of the CLI plugin run sfdx plugins --core)
salesforcedx 49.2.3 (core)
├─ salesforcedx-templates 48.32.0 (core)
├─ custom-metadata 1.0.7 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
└─ salesforce-alm 49.3.0 (core)
**OS and version**:
10.15.2 (19C57) Catalina

**Comment by Confirm4Crit** [comment]
I've got a very sloppy repo of a similar example on a scratch org
https://github.com/Confirm4Crit/ProfilePullRepo/commit/d04f4ef2ae2813364a34ce786c466119b939c502#diff-eda55c3e3fa6fa89b591bc6432cb368d
Field permissions already existed locally, were pushed, and then nuked via a pull.

I recorded a horrible 5 min demo near this with other weird happenings. User permissions not pulling to standard profiles, but pushing fine, etc.
